packages/eicrecon/package.py

```diff
@@ -17,6 +17,10 @@ class Eicrecon(CMakePackage):
     maintainers = ["wdconinc"]
 
     version("main", branch="main")
+    version(
+        "1.6.2",
+        sha256="0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Looks like there is no way to get hash from CI

Reply:
I've been wanting to rewrite the CI job that generates issues for new versions into one that creates PRs for new versions. It's possible to get the hash there.

Reply:
So now I have the CI workflow that creates pull requests, but it can't push them to the repo with a regular GITHUB_TOKEN since that triggers a workflow to run and it doesn't have the permissions for that :roll_eyes: 

See https://github.com/eic/eic-spack/actions/runs/6726195339 and https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#triggering-further-workflow-runs. Anyway, getting closer to having this be more automatic.

Reply:
By hand for now:
```suggestion
        sha256="725d53bc4527486fd15d600b1bd329228502ef83c0ff118cc4600eac7cc2d148",
```